scripts/start-isolated

```diff
@@ -3,10 +3,10 @@
 set -e
 projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; pwd)"
 
-# Check whether mine-support network needs creating
+# Check whether ffc-demo network needs creating
 if [ -z "$(docker network ls --filter name=^mine-support$ --format={{.Name}})" ]; then
```

Comment:
This `if` line checks for a `mine-support` network

Reply:
Good spot.  Fixed.